app/helpers/surveys_helper.rb

```diff
@@ -0,0 +1,138 @@
+module SurveysHelper
```

Comment:
Also, if this is entirely static data, perhaps it might be worth thinking about extracting this into a YAML file

Reply:
Sure thing @Levia I will do this!